src/View/Helper/FormHelper.php

```diff
@@ -1850,7 +1856,7 @@ public function submit($caption = null, array $options = [])
 
         $input = $this->formatTemplate('inputSubmit', [
             'type' => $type,
-            'attrs' => $this->templater()->formatAttributes($options),
+            'attrs' => $this->templater()->formatAttributes($options, ['label']),
```

Comment:
Why is this change necessary?  Submit buttons have nothing to do with labels so that should never be passed to `submit()` options.